README.md

```diff
@@ -26,17 +26,39 @@ In this repository, the processes and features shown in a GitOps process of deve
 
 To be successful in recreating the use cases supported by this pipeline, there are initial steps that should be completed prior to configuring this repository:
 
-- Completion of all pre-requisites and confiuration steps leading to [Feature Development](https://github.com/pingidentity/pipeline-example-platform?tab=readme-ov-file#feature-development) in the example-pipeline-platform.
-- [Docker](https://docs.docker.com/engine/install/) - used to deploy a local sample application.
+- Completion of all pre-requisites and configuration steps leading to [Feature Development](https://github.com/pingidentity/pipeline-example-platform?tab=readme-ov-file#feature-development) from the example-pipeline-platform repository
+- [Docker](https://docs.docker.com/engine/install/) - used to deploy the UI for a sample interface
+- [tflint](https://github.com/terraform-linters/tflint) - for Terraform linting
+- [dvlint](https://github.com/pingidentity/dvlint) - for Davinci flow linting
+- [trivy](https://github.com/aquasecurity/trivy) - for security scanning
+
+> Note - The last three tools are used by the pipeline in Github, and the pipeline will fail if these tests and configuration checks are not passed. It is recommended to install these tools locally to ensure that the pipeline will pass when changes are pushed.
```

Comment:
We do have a `make devcheck` command too and these tools will be run locally. So maybe:

The last three tools are used by the pipeline in Github, and the pipeline will fail if these tests and configuration checks are not passed. Installing these tools locally and using `make devcheck` before committing changes helps ensure that the pipeline will pass when changes are pushed.

Reply:
good catch - had forgot about the **make** target